department-of-veterans-affairs/caseflow
# APPEALS-16188 use remand_appeal for dashboard request issues

Resolves [APPEALS-16188](https://vajira.max.gov/browse/APPEALS-16188)

### Description
- Change cavc_dashboard model and serializer to use remand_appeal for request issues
- Modify CavcDashboardIssuesSection component to use remand_request_issues from serialized dashboard
- Update tests

### Acceptance Criteria

### Testing Plan
1. In demo, switch user to OCC or OAI user
2. Go to any CAVC Remand appeal's case details page (recommend 410000021)
3. Using the "View all cases" button at the top of the page, check that the original appeal stream and CAVC appeal stream have a different number of issues. If they are the same, use a different CAVC remand.
4. Note how many issues are present and their benefit type and issue description
5. Go to the CAVC Dashboard for that appeal
6. Verify that the same issues from the CAVC remand appear in the "Issues" section of the dashboard (NOTE: the "Issue" text on the CAVC dashboard is formatted as "{Appeal Type} - {Issue Description}")
